apps/docs/docs/state/tutorials/basic-tutorial/01-setup.md

```diff
@@ -6,7 +6,7 @@ title: Setting up a Reactive State
 
 # Setting up a Reactive State
 
-We're assuming you've already covered the basics [here](/docs/state/getting-started/setup).
+We're assuming you've already covered the basics [here](/docs/state/setup).
```

Comment:
suggestion: Let's use a relative Markdown link instead. Verify that the following is correct
```suggestion
We're assuming you've already covered the basics [here](../../setup.md).
```